src/session/session_api.c

```diff
@@ -1449,7 +1456,44 @@ __wt_session_range_truncate(
             goto done;
     }
 
-    WT_ERR(__wt_schema_range_truncate(session, start, stop));
+    /*
+     * Create a copy of the start and stop cursors to maintain the original start and stop positions
+     * for error-checking purposes.
+     */
+#ifdef HAVE_DIAGNOSTIC
+    debug_start = debug_stop = NULL;
```

Comment:
The cursor pointers are initialized twice. The initialization at the start is enough.